tpkg/do-tests

```diff
@@ -58,6 +61,10 @@ for i in *.tdir; do
 		echo ">>> skipped $i: test needs pcat, pcat-print, pcat-diff."
 		skip=yes
 	fi
+	if [ $CPU_AFFINITY = no ] && echo $CPU_AFFINITY | grep `basename $i .tdir` >/dev/null; then
```

Comment:
do you mean `echo $NEED_CPU_AFFINITY` that would select the elements of that list for skip.

Reply:
Indeed